with-realm/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+	"compilerOptions": {},
```

Comment:
Format seems off

Reply:
Also we generally just ship examples as JS since expo CLI can auto add TS. Surprisingly, many people are still a bit deterred by TypeScript. Up to you though.

Reply:
I do have both versions of the code samples. I can add instructions in the README on how to add TypeScript and also link to our example code.

Reply:
Add this snippet:
https://github.com/expo/examples/blob/a73c754f560a1aeb1efb9dd03a1b17d78e525269/with-realm/README.md?plain=1#L39-L41